pages/base.py

```diff
@@ -39,35 +39,30 @@ def was_account_created_successfully(self):
     # Not logged in
 
     @property
-    def is_browserid_link_present(self):
-        return self.is_element_present(*self._browserid_login_locator)
+    def is_sign_in_button_present(self):
+        return self.is_element_present(*self._sign_in_button_locator)
 
     @property
     def is_user_loggedin(self):
         return self.is_element_present(*self._logout_locator)
 
-    def click_browserid_login(self):
-        self.selenium.find_element(*self._browserid_login_locator).click()
+    def click_sign_in_button(self):
+        self.selenium.find_element(*self._sign_in_button_locator).click()
 
-    def login(self, email, password):
-        self.click_browserid_login()
-        from browserid import BrowserID
-        pop_up = BrowserID(self.selenium, self.timeout)
-        pop_up.sign_in(email, password)
+    def login(self, email):
+        self.click_sign_in_button()
+        from pages.auth_zero import AuthZero
```

Comment:
I don't know if there is a standard regarding where to place the imports, so I moved it together with the other imports